src/CustomDatePickerAndroid.js

```diff
@@ -73,7 +73,6 @@ export default class CustomDatePickerAndroid extends React.PureComponent {
       }
 
       if (this.props.mode === "datetime") {
-        // Prepopulate and show time picker
```

Comment:
Not sure why remove these two comments but since they doesn't seem too useful I'm fine with your change 👍

Reply:
Yeah, I thought the same 

But I must say, your repository is really well documented. In fact, It helped me & my team demonstrate our product better in a national level competition!

Reply:
Glad to hear that!